models/workitem.go

```diff
@@ -0,0 +1,16 @@
+package models
+
+import (
+)
+
+type WorkItem struct {
+	ID uint `gorm:"primary_key"`
+
+	// User Readable Name of this item
+	Name string 
```

Comment:
What is the intention with Name of a WorkItem? Are you thinking UserStory, Epic etc? If so, I was expecting that to be the 'DisplayName' or similar of the WorkItemType?